rosenpass/src/protocol.rs

```diff
@@ -184,6 +195,89 @@ pub struct CryptoServer {
 
     // Tick handling
     pub peer_poll_off: usize,
+
```

Comment:
I think it should be possible to reuse the code from the biscuit key?

Reply:
not sure I understand, could you elaborate?

Reply:
The responder already issues a type of cookie called a "biscuit" at a later time in the handshake. The biscuit key functions similarly to the CookieKey, also needing to be replaced regularly, also storing a stale key to deal with the period after they key was replaced.

All that logic is already implemented in the BiscuitKey structure.

https://github.com/rosenpass/rosenpass/blob/184cff0e5ec1d67878801b62a91b4d21a679fb20/rosenpass/src/protocol.rs#L449

No need to redo all the work. Lets rename it "CookieKey" and use it for both the biscuit and cookie keys.

Reply:
done, named it `CookieStore`.